app/views/TellUsAboutYourRulingView.scala.html

```diff
@@ -57,22 +61,20 @@
         @caption(messages("caption.goods"))
         @heading(messages("tellUsAboutYourRuling.heading"))
 
+        @hmrcCharacterCount(CharCount(
+            id = "value",
+            name = "value",
+            maxLength = Some(8000),
+            label = Label(
+                content = Text(messages("tellUsAboutYourRuling.label")),
+                classes = "govuk-visually-hidden"
+            ),
+            hint = Some(Hint(content = HtmlContent(content))),
+            value = form("value").value,
+            errorMessage = form("value").error.map(err => ErrorMessage(content = Text(messages(err.message, err.args:_*))))
+        ))
 
 
-        @govukTextarea(
-            TextareaViewModel(
-                field = form("value"),
-                label = Label(
-                    content = Text(messages("tellUsAboutYourRuling.label")),
-                    classes = "govuk-visually-hidden",
-                    )
-                )
-                .withRows(15)
```

Comment:
Now that this is removed the textbox is smaller (on page load). If we change that here then we would have to change it everywhere else for consistency, so is there any way to set it to 15 rows using HmrcCharacterCount?

Reply:
Change made in commit eaf0f764259b0ecf471fdd48e884c1e96badc039